prinfo.json

```diff
@@ -0,0 +1 @@
","name":"Build (linux, arm64)","status":"COMPLETED","conclusion":"SUCCESS","startedAt":"2022-05-18T04:04:44Z","completedAt":"2022-05-18T04:10:09Z","detailsUrl":"https://github.com/juju/juju/runs/6482727595?check_suite_focus=true"},{"__typename":"CheckRun","name":"Build (linux, s390x)","status":"COMPLETED","conclusion":"SUCCESS","startedAt":"2022-05-18T04:04:43Z","completedAt":"2022-05-18T04:10:01Z","detailsUrl":"https://github.com/juju/juju/runs/6482727639?check_suite_focus=true"},{"__typename":"CheckRun","name":"Build (linux, ppc64le)","status":"COMPLETED","conclusion":"SUCCESS","startedAt":"2022-05-18T04:04:44Z","completedAt":"2022-05-18T04:10:03Z","detailsUrl":"https://github.com/juju/juju/runs/6482727683?check_suite_focus=true"},{"__typename":"CheckRun","name":"Build (windows, amd64)","status":"COMPLETED","conclusion":"SUCCESS","startedAt":"2022-05-18T04:04:42Z","completedAt":"2022-05-18T04:09:49Z","detailsUrl":"https://github.com/juju/juju/runs/6482727723?check_suite_focus=true"},{"__typename":"CheckRun","name":"Build (darwin, amd64)","status":"COMPLETED","conclusion":"SUCCESS","startedAt":"2022-05-18T04:04:44Z","completedAt":"2022-05-18T04:09:47Z","detailsUrl":"https://github.com/juju/juju/runs/6482727773?check_suite_focus=true"},{"__typename":"CheckRun","name":"Upgrade (latest/stable, localhost)","status":"COMPLETED","conclusion":"CANCELLED","startedAt":"2022-05-18T04:04:44Z","completedAt":"2022-05-18T04:08:17Z","detailsUrl":"https://github.com/juju/juju/runs/6482727835?check_suite_focus=true"},{"__typename":"CheckRun","name":"Schema","status":"COMPLETED","conclusion":"SUCCESS","startedAt":"2022-05-18T04:04:41Z","completedAt":"2022-05-18T04:08:29Z","detailsUrl":"https://github.com/juju/juju/runs/6482727707?check_suite_focus=true"},{"__typename":"CheckRun","name":"Upgrade (latest/stable, microk8s)","status":"COMPLETED","conclusion":"FAILURE","startedAt":"2022-05-18T04:04:44Z","completedAt":"2022-05-18T04:07:59Z","detailsUrl":"https://github.com/juju/juju/runs/6482727881?check_suite_focus=true"},{"__typename":"CheckRun","name":"Upgrade (latest/beta, localhost)","status":"COMPLETED","conclusion":"SUCCESS","startedAt":"2022-05-18T04:04:44Z","completedAt":"2022-05-18T04:04:50Z","detailsUrl":"https://github.com/juju/juju/runs/6482727928?check_suite_focus=true"},{"__typename":"CheckRun","name":"Upgrade (latest/beta, microk8s)","status":"COMPLETED","conclusion":"SUCCESS","startedAt":"2022-05-18T04:04:44Z","completedAt":"2022-05-18T04:04:49Z","detailsUrl":"https://github.com/juju/juju/runs/6482727971?check_suite_focus=true"},{"__typename":"StatusContext","name":"","context":"check-multi-juju","state":"SUCCESS","status":"","conclusion":"","startedAt":"0001-01-01T00:00:00Z","completedAt":"0001-01-01T00:00:00Z","detailsUrl":"","targetUrl":"https://juju-temp-jenkins.haza.dev/job/github-juju-check-jobs/334/"}],"title":"[JUJU-1057] Fix GitHub Actions on `develop` branch","updatedAt":"2022-05-19T01:09:45Z","url":"https://github.com/juju/juju/pull/14006"}
```

Comment:
Was this supposed to be committed?

Reply:
Oh dear, don't know how that snuck in there. I'll delete it